pkg/mycarehub/application/exceptions/custom_errors.go

```diff
@@ -432,3 +432,12 @@ func GetUserPermissionsErr(err error) error {
 		Code:    int(GetUserPermissionsError),
 	}
 }
+
+// RevokeRolesErr returns an error message when the user roles' revoke fails
```

Comment:
Change to docstring to ``// RevokeRolesErr returns an error message when the user roles' revocation fails``